### PR TITLE
Non-type template parameters can be variadic as well

### PR DESCRIPTION
**Pull request by jdrouhard**

### CODE REVIEW

**Comment by jdrouhard** [comment]
@maxbrunsfeld thanks for merging a couple of the PRs yesterday. Just wanted to make sure you knew these other two were here also!

File: grammar.js [hunk 1]
```diff
@@ -237,6 +238,15 @@ module.exports = grammar(C, {
       optional($._type_identifier)
     )),
 
+    variadic_declaration: $ => prec(1, seq(
```
Comment (maxbrunsfeld) [inline grammar.js]: What do you think of calling this `variadic_parameter_declaration`?

Comment (jdrouhard) [reply grammar.js]: Works for me.

Comment (jdrouhard) [reply grammar.js]: Well, there's already a `variadic_parameter_declaration`, so that doesn't work...

Comment (jdrouhard) [reply grammar.js]: Turns out this new type isn't necessary.

**Comment by jdrouhard** [comment]
@maxbrunsfeld should be good to go again

File: grammar.js (hunk 1 above)
Comment (maxbrunsfeld) [reply grammar.js]: Oh, wonderful. I forgot about that.

**Pull request merged**